### ISSUE TITLE
[Errors] No error trying to import shadowed internal module

### ISSUE THREAD

**Issue by mhegazy** [microsoft/TypeScript#73]
Code, should error but doesn't:

``` TypeScript
    module A {
        export var x;
    }
        module B {
        var A = 1;
// !!! Internal module reference 'A' in import declaration does not reference module instance for 'A'.        
    import Y = A; // Should cause an error
        console.log(Y.x);
    }
```
